### PR TITLE
Add info about a required manual step to graphql-apollo-server recipe

### PR DESCRIPTION
**Pull request by noxify**
Closes: #2904 

### What are the changes and their implications?

Added a new step to the recipe which informs the user to run the `blitz prisma generate` command.

## Bug Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

## Feature Checklist

- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)
- [ ] Documentation added/updated (submit PR to [blitzjs.com repo `canary` branch](https://github.com/blitz-js/blitzjs.com/tree/canary))

### CODE REVIEW

**Review by beerose — COMMENTED** [review]
Thanks for the contribution. 
However, I'm not sure if that's a good approach, because if a user keeps pressing enter, they won't see the message as it disappears after the installation. I think, we can instead add it to the Readme for now

**Comment by flybayer** [comment]
Let's keep this addition and also add it to the recipe README

**Comment by flybayer** [comment]
Actually, the best solution is to run `blitz prisma migrate` automatically. Do we have a recipe step for running arbitrary node code? 🤔

**Comment by noxify** [comment]
I have added the manual step also to the readme.

> Do we have a recipe step for running arbitrary node code?

Currently, there is no step available in the `RecipeBuilder` to do this.

https://github.com/blitz-js/blitz/blob/457015ae5b300753dc64e1032fae4e7c8ab133ee/packages/installer/src/recipe-builder.ts#L8

Sounds like a new feature request for me :)

**Comment by beerose** [comment]
While `runBlitzCommand` is a nice improvement, I think `runCommand` can be useful so that recipes creators have full flexibility. E.g. let's say we have a new recipe adding some library, and as one of the setup steps, users need to run `<new library> init` or something similar. Having `runCommand` and running `<new library> init` inside of the recipe could be nice. As for the `rm -rf` — someone should spot it in the review, and this code should never be merged.

File: recipes/graphql-apollo-server/index.ts [hunk 1]
```diff
@@ -39,4 +39,10 @@ export default RecipeBuilder()
         assignments: [{type: "assignment", key: "provider", value: '"nexus-prisma"'}],
       }),
   })
+  .printMessage({
+    stepId: "manualStep",
+    stepName: "Informs the user about a manual step",
+    explaination: "After the graphql recipe has been installed successfully, the user has to run `blitz prisma generate` manually.",
+    message: "Please run `blitz prisma generate` in your console/terminal to generate the graphql schema."
+  })
```
Comment (beerose) [inline recipes/graphql-apollo-server/index.ts:47]:
```suggestion
  .printMessage({
    stepId: "manualStep",
    stepName: "Informs the user about a manual step",
    message:
      "Please run `blitz prisma generate` in your console/terminal to generate the graphql schema.",
  })
```

**Comment by beerose** [comment]
I'm going to merge this PR, and when we have #2941 implemented, we can add it to this recipe.

**Pull request merged**